pydicom/tests/test_valuerep.py

```diff
@@ -338,8 +474,8 @@ def test_invalid(self, val: float):
     def test_wrong_type(self):
         """Test calling with a string raises an error"""
         with pytest.raises(
-            TypeError,
-            match="'val' must be of type float or decimal.Decimal"
+                TypeError,
+                match="'val' must be of type float or decimal.Decimal"
         ):
```

Comment:
These have not been intentionally, looks like the auto-format did this... I will see if PEP8 likes it.